src/SpotifyApi.NetCore.Tests/LibraryApiTests.cs

```diff
@@ -11,263 +11,170 @@ namespace SpotifyApi.NetCore.Tests
     [TestClass]
     public class LibraryApiTests
     {
-        [TestCategory("Integration")]
-        [TestCategory("User")]
-        [TestMethod]
-        public async Task CheckUserSavedAlbums_AlbumIds_AnyItems()
+        LibraryApi api;
+        string bearerAccessToken;
+
+        public LibraryApiTests()
         {
-            // arrange
             var http = new HttpClient();
             IConfiguration testConfig = TestsHelper.GetLocalConfig();
+            bearerAccessToken = testConfig.GetValue(typeof(string),
+                "SpotifyUserBearerAccessToken").ToString();
             var accounts = new AccountsService(http, testConfig);
+            api = new LibraryApi(http, accounts);
+        }
```

Comment:
Reusing objects in test classes can have unintended consequences. If you want to keep your test code clean, put this code into a method called `Initialize()` with a `[TestInitialize]` attribute. This means the objects get re-initialized on every test, but you can keep your code clean.